### PR TITLE
Add Global Styles

### LINKED ISSUE #15: Add GlobalStyles
# User story
As a **developer**,
I want create a GlobalStyles.css and write all main colors/fonts 
so I have a general template and I don't have to got nuts with CSS.

# Notes / Links

# Acceptance Criteria
- [x] Add color scheme
- [x] Add font Archivo Black 
- [x] Add font Roboto
- [x] Add basic sizing

### PR DESCRIPTION
**Pull request by rheimers**
Fix #15 
- Global Styles created
- Imported to App.js and preview.js
- Fonts added
- Colors added
- Font-sizes added

### CODE REVIEW

**Review by mschmauder-hub — COMMENTED** [review]
lgtm

File: client/src/GlobalStyles.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        html {
+          @font-face {
+            font-family: "ArchivoBlack";
+            src: url(${ArchivoBlack}) format("truetype");
+          }
+          @font-face {
+            font-family: "Roboto";
+            src: url(${Roboto}) format("truetype");
+          }
+          font-family: "Roboto";
```
Comment (manuelpapa) [inline client/src/GlobalStyles.js:34]: I think it is common to list the `font-family` in the `body` section.

**Review by manuelpapa — COMMENTED** [review]
Very clean and logic code! <3 keep going!

File: client/src/GlobalStyles.js (hunk 1 above)
Comment (rheimers) [reply client/src/GlobalStyles.js:34]: will do so! thx

**Pull request merged**